frontend/src/components/CalendarQalendar.vue

```diff
@@ -174,7 +174,7 @@ const processCalendarColorScheme = (calendarTitle, calendarColor) => {
  * @param d - DayJS date object
  * @returns {*}
  */
-const applyTimezone = (d) => dj.utc(d).tz(dj.tz.guess());
+const applyTimezone = (d) => dj(d).utc().tz(dj.tz.guess());
```

Comment:
According to [the docs](https://day.js.org/docs/en/manipulate/utc) we implemented this wrong 😅 So no clue, how this worked before, but should be fixed now!

Reply:
I thought so too! But setting `utc(true)` gives me wrong times (+2, my utc offset) in the calendar... can you test this with your timezone?